src/daemon/messages.go

```diff
@@ -16,6 +17,13 @@ import (
 	"github.com/skycoin/skycoin/src/util/utc"
 )
 
+var (
+	// Every rejection message prefix must start with "RJC" prefix
+	rejectPrefix = [...]byte{82, 74, 67}
```

Comment:
`RJCT` is a generic message to reject other messages of any kind . This is not needed anymore .

Reply:
New message will break the connection with node that can't recognize the message, if the `RJCT` is not used anymore, we'd better remove it.

Reply:
@iketheadore this is exactly the intention . Every node receiving a `RJCT` message has to disconnect from peer that sent it , which is consistent with current behavior as you just said .

Reply:
`RJCT` can help with other situation I've noticed recently happening quite often . This message notifies of disconnection right away , so peer is removed of PEX connection pool immediately. Now we are not doing this , so many entries are kept to other peers that already disconnected but node is unaware of .

This is a typical log entry reflecting this (obfuscated text is IP:port pairs)

```
[2018-08-18T04:29:50Z] WARN [daemon]: Failed to send *daemon.IntroductionMessage to ###.###.###.###:#####: set tcp ###.###.###.###:####: use of closed network connection
```

This solution will fix that since both ends will be aware of disconnections , thus allowing for graceful disconnection.